tests/test_tasker.py

```diff
@@ -236,7 +238,6 @@ def test_pull_image():
     remote_img.registry = LOCALHOST_REGISTRY
     t.tag_and_push_image(local_img, remote_img, insecure=True)
     got_image = t.pull_image(remote_img, insecure=True)
-    assert remote_img.to_str() == got_image
```

Comment:
Not sure why this check was removed - it seem valid to me

Reply:
it was removed before, when I changed return value of pull_image as it wasn't used anyway, I will return this assert